src/shared/components/modlog.tsx

```diff
@@ -585,106 +751,113 @@ export class Modlog extends Component<any, ModlogState> {
-                    Adding Mod to Community
-                  </option>
-                  <option value={ModlogActionType.ModTransferCommunity}>
-                    Transfering Communities
-                  </option>
-                  <option value={ModlogActionType.ModAdd}>
-                    Adding Mod to Site
-                  </option>
-                  <option value={ModlogActionType.ModBan}>
-                    Banning From Site
-                  </option>
-                </select>
-              </div>
-              {!this.state.siteRes.site_view.local_site
-                .hide_modlog_mod_names && (
-                <div className="form-group col-sm-6">
-                  <select
-                    id="filter-mod"
-                    className="form-control"
-                    value={this.state.filter_mod}
-                  >
-                    <option>{i18n.t("filter_by_mod")}</option>
-                  </select>
-                </div>
-              )}
-              <div className="form-group col-sm-6">
-                <select
-                  id="filter-user"
-                  className="form-control"
-                  value={this.state.filter_user}
-                >
-                  <option>{i18n.t("filter_by_user")}</option>
-                </select>
-              </div>
-            </div>
-            <div className="table-responsive">
+          <div className="form-row">
+            <select
+              value={actionType}
+              onChange={linkEvent(this, this.handleFilterActionChange)}
+              className="custom-select col-sm-6"
+              aria-label="action"
+            >
+              <option disabled aria-hidden="true">
+                {i18n.t("filter_by_action")}
+              </option>
+              <option value={ModlogActionType.All}>{i18n.t("all")}</option>
+              <option value={ModlogActionType.ModRemovePost}>
+                Removing Posts
+              </option>
+              <option value={ModlogActionType.ModLockPost}>
+                Locking Posts
+              </option>
+              <option value={ModlogActionType.ModFeaturePost}>
+                Featuring Posts
+              </option>
+              <option value={ModlogActionType.ModRemoveComment}>
+                Removing Comments
+              </option>
+              <option value={ModlogActionType.ModRemoveCommunity}>
+                Removing Communities
+              </option>
+              <option value={ModlogActionType.ModBanFromCommunity}>
+                Banning From Communities
+              </option>
+              <option value={ModlogActionType.ModAddCommunity}>
+                Adding Mod to Community
+              </option>
+              <option value={ModlogActionType.ModTransferCommunity}>
+                Transferring Communities
+              </option>
+              <option value={ModlogActionType.ModAdd}>
```

Comment:
These should probably have translations.

Reply:
Don't worry about that for now. #31